src/equations/compressible_euler_1d.jl

```diff
@@ -593,6 +593,21 @@ end
   return SVector(w1, w2, w3)
 end
 
+@inline function entropy2cons(w, equations::CompressibleEulerEquations1D)
+  γ = equations.gamma
+
+  w1,wU1,wE = w .* (γ-1) # convert to entropy -ρ*s / (γ-1)
+  
+  wUnorm    = wU1^2
+  s = γ - w1 + wUnorm/(2*wE)
+
+  ρι     = ((γ-1) / (-wE)^γ)^(1/(γ-1))*exp(-s/(γ-1))
+  rho    = -ρι * wE
+  rhov   = ρι * wU1
+  rho_e  = ρι*(1-wUnorm/(2*wE))
+  return SVector(rho,rhov,rho_e)
+end
```

Comment:
I have to admit that I have a hard time verifying this code since some names are rather counter-intuitive for me. For example, I would have expected that `ρι` is some density (`ρ` is  a`rho`, not a `p`). However, `ρι` is actually the pressure divided by `γ - 1` (if I didn't make a mistake). Would you mind changing the names of some variables a bit to make this a bit easier to follow? If you want, I could also make some suggestions.

Reply:
No problem - The variable choices are directly taken from the Hughes, Mallet, Franca paper on entropy variables, so agreed they're a bit opaque. I can try to mimic the names in other functions for this.

Reply:
If you took that from a paper, you can also use the variable names used there and cite that paper (name of the authors, year, title, doi) as a comment

Reply:
That's actually something we try to do whenever we take something from a paper.

Reply:
@ranocha I'd welcome some suggestions for variable names too.

Reply:
Changes made in https://github.com/trixi-framework/Trixi.jl/pull/504/commits/e8876197d238631eb625dee7778a29441139a9e1

Reply:
```suggestion
@inline function entropy2cons(w, equations::CompressibleEulerEquations1D)
  # See Hughes, Franca, Mallet (1986) A new finite element formulation for CFD
  # [DOI: 10.1016/0045-7825(86)90127-1](https://doi.org/10.1016/0045-7825(86)90127-1)
  @unpack gamma = equations

  # convert to entropy `-rho * s` used by Hughes, France, Mallet (1986)
  # instead of `-rho * s / (gamma - 1)`
  V1, V2, V5 = w * (gamma - 1)

  # specific entropy, eq. (53)
  s = gamma - V1 + 0.5 * (V2^2) / V5

  # eq. (52)
  energy_internal = ((gamma - 1) / (-V5)^gamma)^inv(gamma - 1) * exp(-s / (gamma - 1))

  # eq. (51)
  rho    = -V5 * energy_internal
  rho_v1 = V2 * energy_internal
  rho_e  = (1 - 0.5 * (V2^2) / V5) * energy_internal
  return SVector(rho, rho_v1, rho_e)
end
```

Reply:
I see - thanks for the example! 

I can't directly commit the suggestion - is that because I added another commit and "outdated" the old suggestion?

Reply:
Yes. I added it again below